src/SumatraPDF.cpp

```diff
@@ -2001,30 +2002,18 @@ static void RerenderEverything()
 
 void GetFixedPageUiColors(COLORREF& text, COLORREF& bg)
 {
-    if (gGlobalPrefs->useSysColors) {
-        text = GetSysColor(COLOR_WINDOWTEXT);
-        bg = GetSysColor(COLOR_WINDOW);
-    }
-    else {
-        text = gGlobalPrefs->fixedPageUI.textColor;
-        bg = gGlobalPrefs->fixedPageUI.backgroundColor;
-    }
+    

```

Comment:
I was imagining a menu item like `Settings > Theme > Theme Document Colors`. A pre-release seems like a good idea. It will also give the opportunity to modify the current theme colours, especially light, which I just left exactly as it was before and could be improved on (like you were saying it is hard to tell which tab is the active tab in that theme).